framework/captcha/drivers/Driver.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace yii\captcha\drivers;
+
+use yii\base\Object;
+
+/**
+ * Base class for Drivers.
+ */
+abstract class Driver extends Object
+{
+    /**
+     *
+     * @var string[]
+     */
+    private $errors = [];
+
+    /**
+     * Renders the CAPTCHA image.
+     * @param string $code Captcha code
+     * @param ImageSettings $imageSettings Image generate settings.
+     */
+    abstract public function renderCaptcha($code, ImageSettings $imageSettings);
```

Comment:
Since this method will be the only method left, I suggest introducing `DriverInterface` and removal of `Driver` class.